### DISCUSSION TITLE
微信小程序抓包

### DISCUSSION THREAD

**Discussion by Hiwoniu** [wanghongenpin/proxypin#13] [General]
希望尽快支持

**Comment by wanghongenpin** [comment]
安卓后面会先只做微信抓包，通用的实现起来会麻烦点

**Reply by Hiwoniu** [reply]
好的，非常期待。作者加油

**Comment by wanghongenpin** [comment]
安卓微信apk，可以抓小程序包，需要配置wifi代理用电脑版
链接: https://pan.baidu.com/s/1AwbiEgRSMArGRRikIH1hng?pwd=si2s

**Reply by wanghongenpin** [reply]
安卓无ROOT使用Xposed模块抓包: https://github.com/wanghongenpin/network_proxy_flutter/wiki/%E5%AE%89%E5%8D%93%E6%97%A0ROOT%E4%BD%BF%E7%94%A8Xposed%E6%A8%A1%E5%9D%97%E6%8A%93%E5%8C%85
